access/templates/access/login.html

```diff
@@ -17,7 +17,7 @@ <h1>Login</h1>
         const data = new FormData(e.target);
 
         const xml = new XMLHttpRequest();
-        xml.open("POST", "/login");
+        xml.open("POST", {% url 'login' %});
```

Comment:
No quotes around the URL even though this is JavaScript code?

The output JavaScript looks like this: `xml.open("POST", /login);`
As you can see, the string literal `/login` is missing the quotes `" "`. This causes a syntax error and the JS crashes.

I think this should work like this:
`xml.open("POST", "{% url 'login' %}");`
I just added quotes around the output of the `url` template tag.